frontend/src/app/components/filters/quick-filter-by-text-input/quick-filter-by-text-input.component.ts

```diff
@@ -26,25 +26,25 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {Component, OnDestroy, Output} from '@angular/core';
+import {Component, OnDestroy, Output, EventEmitter} from '@angular/core';
 import {I18nService} from "app/modules/common/i18n/i18n.service";
 import {WorkPackageViewFiltersService} from "core-app/modules/work_packages/routing/wp-view-base/view-services/wp-view-filters.service";
-import {componentDestroyed, untilComponentDestroyed} from "ng2-rx-componentdestroyed";
+import {untilComponentDestroyed} from "ng2-rx-componentdestroyed";
 import {WorkPackageCacheService} from "app/components/work-packages/work-package-cache.service";
-import {merge, Observable, Subject} from "rxjs";
-import {debounceTime, delay, delayWhen, distinctUntilChanged, map, startWith, tap} from "rxjs/operators";
-import {DebouncedEventEmitter} from "core-components/angular/debounced-event-emitter";
+import {Subject} from "rxjs";
+import {debounceTime, distinctUntilChanged, map, tap} from "rxjs/operators";
 import {IsolatedQuerySpace} from "core-app/modules/work_packages/query-space/isolated-query-space";
 import {QueryFilterInstanceResource} from "core-app/modules/hal/resources/query-filter-instance-resource";
 import {input} from "reactivestates";
+import {QueryFilterResource} from "core-app/modules/hal/resources/query-filter-resource";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/860cdc9b13e16aeb5d2fc45b77498c48a0282a79/frontend/src/app/components/filters/quick-filter-by-text-input/quick-filter-by-text-input.component.ts#L40"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>